platforms/chibios/drivers/uart.c

```diff
@@ -28,12 +32,17 @@ void uart_init(uint32_t baud) {
 
         serialConfig.speed = baud;
 
-#if defined(USE_GPIOV1)
-        palSetLineMode(SD1_TX_PIN, PAL_MODE_ALTERNATE_OPENDRAIN);
-        palSetLineMode(SD1_RX_PIN, PAL_MODE_ALTERNATE_OPENDRAIN);
+#if defined(WB32F3G71xx)
+        palSetLineMode(SD1_TX_PIN, PAL_MODE_ALTERNATE(SD1_TX_PAL_MODE) | PAL_WB32_PUPDR_PULLUP);
+        palSetLineMode(SD1_RX_PIN, PAL_MODE_ALTERNATE(SD1_TX_PAL_MODE) | PAL_WB32_PUPDR_PULLUP);
 #else
-        palSetLineMode(SD1_TX_PIN, PAL_MODE_ALTERNATE(SD1_TX_PAL_MODE) | PAL_OUTPUT_TYPE_OPENDRAIN);
-        palSetLineMode(SD1_RX_PIN, PAL_MODE_ALTERNATE(SD1_RX_PAL_MODE) | PAL_OUTPUT_TYPE_OPENDRAIN);
+#    if defined(USE_GPIOV1)
+        palSetLineMode(SD1_TX_PIN, PAL_MODE_STM32_ALTERNATE_OPENDRAIN);
+        palSetLineMode(SD1_RX_PIN, PAL_MODE_STM32_ALTERNATE_OPENDRAIN);
+#    else
+        palSetLineMode(SD1_TX_PIN, PAL_MODE_ALTERNATE(SD1_TX_PAL_MODE) | PAL_STM32_OTYPE_OPENDRAIN);
+        palSetLineMode(SD1_RX_PIN, PAL_MODE_ALTERNATE(SD1_RX_PAL_MODE) | PAL_STM32_OTYPE_OPENDRAIN);
+#    endif
```

Comment:
Should be reverted as reintroducing the STM32_defines is wrong.

Instead `PAL_WB32_PUPDR_PULLUP` should be defined to `PAL_OUTPUT_TYPE_OPENDRAIN` within `tmk_core/common/chibios/chibios_config.h` per the other compatibility defines.